services/payments/src/simcore_service_payments/models/payments_gateway.py

```diff
@@ -0,0 +1,55 @@
+from datetime import datetime
+from decimal import Decimal
+from typing import Literal, TypeAlias
+from uuid import UUID
+
+from pydantic import BaseModel
+
+
+class InitPayment(BaseModel):
+    amount_dollars: Decimal
```

Comment:
We decided to stick with dollars and intentionally hard-code this in the "money" variables. Supporting any currency will unnecessarily add a complexity that we do not really need at the moment (since we will probably only deal with dolalrs)